x/fundraising/keeper/auction.go

```diff
@@ -326,13 +358,17 @@ func (k Keeper) AddAllowedBidders(ctx sdk.Context, auctionId uint64, bidders []t
 		return err
 	}
 
+	// Call the before allowed bidders added hook
```

Comment:
Shouldn't this addition go to right before the line of 372?

Reply:
As long as it is located after the end of validation and before updating states, i think it is fine. However, to make it consistent with other hooks, I will make that change.